Tella/Scenes/Settings/Views/Servers/Uwazi/Step4/UwaziTwoStepVerification.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  UwaziTwoStepVerification.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 4/25/23.
+//  Copyright © 2023 INTERNEWS. All rights reserved.
+//
+
+import SwiftUI
+
+struct UwaziTwoStepVerification: View {
+
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    @EnvironmentObject var serverViewModel : UwaziServerViewModel
+    @EnvironmentObject var serversViewModel : ServersViewModel
+
+    var body: some View {
+        ContainerView {
+            VStack {
+                VStack(spacing: 10) {
+                    Spacer()
+                    TopServerView(title: LocalizableSettings.UwaziTwoStepTitle.localized)
+                    Text(LocalizableSettings.UwaziTwoStepMessage.localized)
+                        .font(.custom(Styles.Fonts.regularFontName, size: 14))
+                        .foregroundColor(.white)
+                        .multilineTextAlignment(.center)
+                    Spacer()
+                        .frame(height: 15)
+                    TextfieldView(fieldContent: $serverViewModel.code,
+                                  isValid: $serverViewModel.validCode,
+                                  shouldShowError: $serverViewModel.shouldShowAuthenticationError,
+                                  errorMessage: serverViewModel.codeErrorMessage,
+                                  fieldType: .code,
+                                  placeholder: LocalizableSettings.UwaziAuthenticationPlaceholder.localized)
+                                .keyboardType(.numberPad)
```

Comment:
In my opinion that it was better to manage `keyboardType` inside the TextfieldView depending of `fieldType`

Reply:
Done

Reply:
I was thinking about when we have a password -> .alphabet and when we have code password : .numberPad
but it will be fixed later